logs/es/es.go

```diff
@@ -38,10 +41,13 @@ func (el *esLogger) Init(jsonconfig string) error {
 		return err
 	} else if u.Path == "" {
 		return errors.New("missing prefix")
-	} else if host, port, err := net.SplitHostPort(u.Host); err != nil {
-		return err
 	} else {
-		conn := goes.NewClient(host, port)
+		conn, err := elasticsearch.NewClient(elasticsearch.Config{
+			Addresses: []string{u.Host},
```

Comment:
must be `el.DSN` => Addresses: []string{el.DSN},
Otherwise, you will got
```
unable to WriteMsg to adapter:es,error:unsupported protocol scheme "localhost"
```